### DISCUSSION TITLE
How to implement an Addon with multiple tabs like a11y

### DISCUSSION THREAD

**Discussion by WalkingOS** [storybookjs/storybook#12740] [Help] [answered]
Hey..

I want to show for each component the html, fluid and twig code.

My Idea was to implement an addon with multiple tabs.

I checked the Source Code from a11y but don't understand how it generates the tabs.

Can some body help me?

(For testing purposes my components are written with .pug, .scss and .stories.mdx)

**Comment by ghost** [comment, marked as answer]
No exactly what you're looking for, but I recently found a third party addon called [storybook-addon-preview](https://www.npmjs.com/package/storybook-addon-preview) that can be used to show source code in different tabs the way you want it.

**Comment by tooppaaa** [comment]
Addon-A11y Panel is simply composed of tabs in it's main UI component. It's not part of Storybook's addons.
